source/catalog.html

```diff
@@ -202,71 +202,79 @@ <h3>Cat Energy SLIM 500 г</h3>
                 </tr>
               </table>
               <a class="button product-card__button">заказать</a>
-            </article>
+              </article>
             </li>
             <li class="catalog__item">
-              <a class="product-card__more-icon" href="#!">
-                <svg width="30" height="30" fill="none"><path stroke="#D3D3D3" stroke-width="2" d="M15 0v30M30 15H0"/></svg>
-              </a>
-              <p class="product-card__description">Показать еще 100500 товаров</p>
-              <p class="product-card__text">На самом деле вкусов гораздо больше!</p>
-              <a class="button product-card__button-more" href="#!">показать все</a>
+              <div class="product-card__more">
+                <a class="product-card__more-icon" href="#!">
```

Comment:
нарушение БЭМ. Использование бем-елемента без самого бем-блока https://up.htmlacademy.ru/adaptive/24/criteries#b5
в этой лишке используй классы, относящиеся к блоку catalog
```suggestion
              <div class="catalog__more">
                <a class="catalog__more-icon" href="#!">
```